src/control/cmd/daos/util.go

```diff
@@ -240,6 +241,8 @@ func allocCmdArgs(log logging.Logger) (ap *C.struct_cmd_args_s, cleanFn func(),
 		outCleanup()
 		errCleanup()
 		freeCmdArgs(ap)
+		// Give the streams a chance to flush.
+		time.Sleep(250 * time.Millisecond)
```

Comment:
Instead of sleeping, is it not possible to check if the streams have been flushed or not, or to flush them synchronously ?